libraries/lib-wave-track/WaveClip.cpp

```diff
@@ -664,7 +664,14 @@ bool WaveClip::Paste(double t0, const WaveClip& other)
    if (GetWidth() != other.GetWidth())
       return false;
 
-   assert(GetStretchRatio() == other.GetStretchRatio());
+   if (GetSequenceSamplesCount() == 0)
+   {
+      // Empty clip: we're flexible and adopt the other's stretching.
+      mRawAudioTempo = other.mRawAudioTempo;
+      mClipStretchRatio = other.mClipStretchRatio;
```

Comment:
Must you also copy `mProjectTempo` to be sure `GetStrectchRatio` returns the same for both clips?

Reply:
I don't expect there to be a use case for that: an empty clip normally is something freshly created for future insertion into a track, at which stage it gets the project tempo update.
But it wouldn't hurt to be extra cautious. Will add a to-be-squashed commit for that.